README.md

```diff
@@ -26,153 +26,166 @@
- let g:dashboard_custom_shortcut_icon['book_marks'] = ' '
+  Note: There is one extra space after the icon character to improve visibility.
 
-```
+  ```vim
+  let g:dashboard_custom_shortcut_icon['last_session'] = ' '
+  let g:dashboard_custom_shortcut_icon['find_history'] = 'ﭯ '
+  let g:dashboard_custom_shortcut_icon['find_file'] = ' '
+  let g:dashboard_custom_shortcut_icon['new_file'] = ' '
+  let g:dashboard_custom_shortcut_icon['change_colorscheme'] = ' '
+  let g:dashboard_custom_shortcut_icon['find_word'] = ' '
+  let g:dashboard_custom_shortcut_icon['book_marks'] = ' '
+  ```
 
-- what does the shortcut do? just a tip like `whichkey`,on dashboard you just
-  move the cursor and press `enter`
+- What does the shortcut do? 
 
-- `g:dashboard_custom_header` custom the dashboard header (same as startify)
-  check [wiki](https://github.com/glepnir/dashboard-nvim/wiki/Ascii-Header-Text)
-  to find more Ascii Text Header collection.
+    Just a tip like `whichkey`, on the dashboard, shortcuts are items that you can use the cursor to navigate around and press `Enter` to confirm.
 
-- `g:dashboard_custom_footer` custom the dashboard footer (same as startify)
+- `g:dashboard_custom_header` customs the dashboard header (same as startify). Check [wiki](https://github.com/glepnir/dashboard-nvim/wiki/Ascii-Header-Text) to find more Ascii Text Header collections.
+  
+- `g:dashboard_custom_footer` customs the dashboard footer (same as startify).
 
-- Dashboard provide session support with `SessionLoad` and `SessionSave`
-  commands you can define keymap like this .
-  ```viml
+- Dashboard provides session support. With `SessionLoad` and `SessionSave` commands, you can define keymaps like below:
+  
+  ```vim
   nmap <Leader>ss :<C-u>SessionSave<CR>
   nmap <Leader>sl :<C-u>SessionLoad<CR>
   ```
-  set the `dashboard_session_directory` to change the session folder
-  default is `~/.cache/vim/session`
-- Highlight group
-  ```VimL
+  Set `dashboard_session_directory` to change the session folder which by default is `~/.cache/vim/session`.
+  
+- Highlight groups
+  ```vim
   DashboardHeader
   DashboardCenter
   DashboardShortcut
   DashboardFooter
   ```
-- Autocmd `Dashboard` `DashboardReady` same as vim-startify
 
-- `g:dashboard_custom_section` custom section, it's a dict type,key is your section component name,
-  It will be used to sort. every component is a dict and must have `description` and `command`,
-  `description` is a list that show in Dashboard buffer,`command` is string or funcref type.
+- Autocmd `Dashboard` `DashboardReady` are the same as vim-startify.
+
+- `g:dashboard_custom_section` customs your own sections. 
+  
+  It's a dictionary whose entries will be your DIY shortcuts.
 
-  ```viml
+  Each entry will be an object that must contain attribute `description` and attribute `command`:
+  
+  * `description` is a string shown in Dashboard buffer.
+  * `command` is a string or funcref type.
```

Comment:
add `you can pass every command from fuzzy find plugin which you used?` will be better?

Reply:
```suggestion
  * `command` is a string or funcref type. You can pass every command expended from the fuzzy find plugins which are available locally.
```
What do you think?